integration-test/common/src/test/java/org/apache/uniffle/test/QuorumTest.java

```diff
@@ -101,6 +106,15 @@ public static MockedShuffleServer createServer(int id, File tmpDir, int coordina
     return new MockedShuffleServer(shuffleServerConf);
   }
 
+  private Integer generateFakePort(Set<Integer> portExistsSet) {
```

Comment:
Can a int array of size 5 be directly generated, with values ranging from 1 to 65535, and ensuring that this array has no intersection with `portExistsSet`?

Reply:
done